tools/argdist.py

```diff
@@ -200,28 +372,42 @@ def display(self):
         Print a frequency count of how many times process 1005 called malloc
         with an allocation size of 16 bytes
 
-argdist.py -C 'r:c:gets():char*:$retval;snooped strings'
+argdist.py -C 'r:c:gets():char*:$retval#snooped strings'
         Snoop on all strings returned by gets()
 
-argdist.py -p 1005 -C 'p:c:write(int fd):int:fd'
+argdist.py -H 'r::__kmalloc(size_t size):u64:$latency/$entry(size)#ns per byte'
+        Print a histogram of nanoseconds per byte from kmalloc allocations
+
+argdist.py -I 'linux/slab.h' \\
```

Comment:
This swiss army knife looks great, but -I flag is too hard for the average user who won't know which header defines gfp_t. We can either include a lot of headers by default or parse the command line for known symbols and add .h automatically. I guess we can keep -I for obscure cases of driver debugging, but things mentioned in the example file (like gfp_t and timespec) should be automatic.